content/contributing/opencue/getting-started-windows.md

```diff
@@ -0,0 +1,186 @@
+---
+title: "Development Environment setup on Windows"
+linkTitle: "Development Environment setup on Windows"
+date: 2020-01-29
+weight: 2
+description: >
+  A guide to setting up an end-to-end development environment on Windows.
+---
+
+<!---DO NOT MODIFY THIS LINE--->
+
+This is a guide to setting up a development environment on Windows that will allow 
+you to run the entire OpenCue system from end-to-end (Cuebot, Cuesubmit, CueGUI, RQD)
+and make changes to any part of it.
+
+## Prerequisites
```

Comment:
Are there any minimum version numbers we should list for this software? For example, I think it's version 9 or greater of Postgres.

Reply:
Have added suggested versions.